timesys/high-level-triggers.py

```diff
@@ -696,10 +696,22 @@
             'NrPulses': {'value': 1, 'hilim': 2, 'high': 2, 'hihi': 2},
             'Duration': {'value': 0.3, 'hilim': 1.6, 'high': 1.6, 'hihi': 1.6},
             'State': {'value': 0},
-            'Polarity': {'value': 1, 'states': (0, 2)},
+            'Polarity': {'value': 0, 'states': (0, 2)},
```

Comment:
Silly question. Is the polarity value correct here?

Reply:
Looking at the last saved config, LI-01:TI-EGun-MultBun:Polarity-Sel and LI-01: TI-EGun-SglBun: Polarity-Sel are 0 indeed, so I thing it is correct!

Reply:
Ok then. Nice.